cmd/dra-example-kubeletplugin/driver.go

```diff
@@ -101,6 +101,9 @@ func (d *driver) Shutdown(logger klog.Logger) error {
 
 func (d *driver) PrepareResourceClaims(ctx context.Context, claims []*resourceapi.ResourceClaim) (map[types.UID]kubeletplugin.PrepareResult, error) {
 	klog.Infof("PrepareResourceClaims is called: number of claims: %d", len(claims))
+	for i, claim := range claims {
+		klog.Infof("Claim %d: UID=%s, Namespace=%s, Name=%s", i, claim.UID, claim.Namespace, claim.Name)
```

Comment:
Could we add this log to `prepareResourceClaim` so we don't need this extra loop?